addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/hardware/FritzahaWebInterface.java

```diff
@@ -300,12 +315,49 @@ public FritzahaContentExchange asyncPost(String path, String args, FritzAhaCallb
         if (!isAuthenticated()) {
             authenticate();
         }
-
         FritzahaContentExchange postExchange = new FritzahaContentExchange(callback);
-
-        this.asyncclient.newRequest(this.getURL(path)).timeout(this.config.getAsyncTimeout(), TimeUnit.SECONDS)
+        asyncclient.newRequest(getURL(path)).timeout(this.config.getAsyncTimeout(), TimeUnit.SECONDS)
                 .method(HttpMethod.POST).onResponseSuccess(postExchange).onResponseFailure(postExchange) // .onComplete(postExchange)
-                .content(new StringContentProvider(this.addSID(args), "UTF-8")).send(postExchange);
+                .content(new StringContentProvider(addSID(args), "UTF-8")).send(postExchange);
         return postExchange;
     }
+
+    public FritzahaContentExchange getTemperature(String ain) {
+        return null;
```

Comment:
Referring to the FRITZ! AHA documentation it ist possible to request the values for those instance. But two facts forced me to stop my research and development here:
- The FRITZ! AHA interface endpoints won't deliver well-formed content. It is neither XML nor another standard which can be easily processed.
- The structure of the binding itself is IMHO not very reusable. There will be a lot work for little reward.

I like to refactor it comnpletely in the future.